app/services/importer/get_greenhouse_fields.rb

```diff
@@ -17,6 +17,13 @@ def initialize(job, data)
 
     private
 
+    def build_education_question(type)
+      puts "Building education question, type: #{type}."
+      question = @standard_fields[type].merge(required: @education_required)
+      question[:fields][0][:options] = Importer::GreenhouseEducationOptionsFetcher.call(@company_id, type) if options?(question[:fields][0])
+      question
```

Comment:
Shouldn't it be cleaner if the Fetcher was checking the condition in the processable method ?